tests/string/CreditCard.test.ts

```diff
@@ -0,0 +1,59 @@
+import * as CreditCard from '../../src/string/CreditCard'
```

Comment:
🔧  I think it'd be worth it to add these test cases from validators.js:

https://github.com/validatorjs/validator.js/blob/86a07ba4f3f710f639e92a62cf81dd3321ef9ee8/test/validators.js#L5089